client/screens/feed/Feed.tsx

```diff
@@ -74,72 +69,70 @@ const FeedSearchFilter = ({
   const styles = useCustomStyles(loadStyles);
   return (
     <View style={styles.filterContainer}>
-      <Box style={styles.searchContainer}>
-        <HStack space={3}>
-          <Input
-            w="80%"
-            variant="outline"
+      <View style={styles.searchContainer}>
+        <RStack space={3} style={{flexDirection: 'row', justifyContent: 'center'}}>
+          <RInput
+            size="$30"
             placeholder={`Search ${feedType || 'Feed'}`}
             onChangeText={setSearchQuery}
           />
-          <IconButton
+          <RIconButton
+            backgroundColor="transparent"
             icon={
               <AntDesign
                 name="search1"
                 size={24}
                 color={currentTheme.colors.cardIconColor}
               />
             }
-            variant="ghost"
           />
-        </HStack>
-      </Box>
-      <Divider my={3} />
-      <Center
-        space={3}
+        </RStack>
+      </View>
+      <RSeparator />
+      <RStack
+        flex={1}
+        flexWrap="wrap"
         flexDirection="row"
         justifyContent="space-between"
         alignItems="center"
-        flexWrap={'wrap'}
         padding={2}
         margin={2}
       >
         {feedType === 'public' && (
-          <HStack space={3} alignItems="center">
-            <Text
-              fontSize="lg"
+          <RStack style={{flexDirection: 'row', gap: '10px', alignItems: 'center'}}>
+            <RText
+              fontSize={18}
               fontWeight="bold"
               color={currentTheme.colors.textColor}
             >
               Packs
-            </Text>
-            <Switch
-              size="lg"
-              isChecked={selectedTypes.pack}
-              onToggle={handleTogglePack}
+            </RText>
+            <RSwitch
+              size="$1.5"
+              checked={selectedTypes.pack}
+              onCheckedChange={handleTogglePack}
             />
-            <Text
-              fontSize="lg"
+            <RText
+              fontSize={18}
               fontWeight="bold"
               color={currentTheme.colors.textColor}
             >
               Trips
-            </Text>
-            <Switch
-              size="lg"
-              isChecked={selectedTypes.trip}
-              onToggle={handleToggleTrip}
+            </RText>
+            <RSwitch
+              size="$1.5"
+              checked={selectedTypes.trip}
+              onCheckedChange={handleToggleTrip}
             />
-          </HStack>
+          </RStack>
         )}
-        <HStack space={3} alignItems="center">
-          <Text
-            fontSize="lg"
-            fontWeight="bold"
```

Comment:
we might want bold still

Reply:
Thanks for the piece of feedback! I made that call while refactoring the design as I found the non-bold version closer to the apperance of the original view. I will put back the `bold` and prioritize keeping the HTML/design semantic over visual fidelity.

Reply:
I like your thinking! that's a good thing to be aware of